### PR TITLE
Lets get to it, Travis!

### PR DESCRIPTION
**Pull request by CodeBaboon**
Time to get things all legit 'n stuff

### CODE REVIEW

**Comment by ryantmer** [comment]
Yay Travis worked! Travis is broken, but Travis worked. #ThingsThatMakeSenseInSoftware

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js: stable
+script:
+- npm run test:lint
+- wct --skip-plugin local
+addons:
+  sauce_connect: true
+  jwt:
```
Comment (ryantmer) [inline .travis.yml]: Poking around at other repos I see two ways of implementing this, either

```
env:
  global:
    - secure: <some encrypted sauce token here I guess>
addons:
  sauce_connect: true
```

or

```
addons:
  sauce_connect:
    username:
      secure: <some encrypted username here I guess>
    access_key:
      secure: <some encrypted access key here I guess>
```

Is what you've got here another way of doing the same thing? Not familiar enough with this stuff, sorry :/

Comment (CodeBaboon) [inline .travis.yml]: I copied what I have from the `ajax` repo, which seems to run tests? I know the secure value I had in the first commit was invalid, I have just updated with a newly encypted one, so we'll see if that works. If not I'll try the formats you've got above.

Comment (CodeBaboon) [inline .travis.yml]: Trav is still angry. Next attempt...

**Comment by CodeBaboon** [comment]
1 out of 2 ain't half bad...

**Comment by CodeBaboon** [comment]
Presumably there is a chance that this just needs to get merged into master before the PR builds will succeed? From what I've read the Push builds are based on the last commit of the current branch, whereas the PR build is a merge of this branch and master. The branch is up to date with master so it seems like both scenarios should work fine, but for some reason the sauce credentials aren't being applied correctly on the PR builds? I don't see a lot of harm in merging this PR and then back-filling a fix afterwards if it still doesn't work.

**Pull request merged**